core/tests/protractor/collections.js

```diff
@@ -20,14 +20,21 @@ var general = require('../protractor_utils/general.js');
 var users = require('../protractor_utils/users.js');
 var AdminPage = require('../protractor_utils/AdminPage.js');
 var collectionEditor = require('../protractor_utils/collectionEditor.js');
+var CreatorDashboardPage =
+  require('../protractor_utils/CreatorDashboardPage.js');
 
 
 describe('Collections', function() {
   var adminPage = null;
+  var creatorDashboardPage = null;
+  var COLLECTION_ID = null;
```

Comment:
collectionId -- this gets modified so it's not a constant.

Reply:
Ok, changed that.